WcaOnRails/config/routes.rb

```diff
@@ -42,6 +42,11 @@
     get 'register' => 'registrations#register'
     get 'register-require-sign-in' => 'registrations#register_require_sign_in'
   end
+
+  get 'competitions/:competition_id/report/edit' => 'delegate_reports#edit', as: :delegate_report_edit
+  get 'competitions/:competition_id/report' => 'delegate_reports#show', as: :delegate_report
+  patch 'competitions/:competition_id/report' => 'delegate_reports#update'
```

Comment:
I see. That does seem to do *something*, but not exactly what we want.

With your suggestion:

```
                             edit_delegate_report GET    /competitions/:competition_id/report/:id/edit(.:format)                     delegate_reports#edit
                                  delegate_report GET    /competitions/:competition_id/report/:id(.:format)                          delegate_reports#show
                                                  PATCH  /competitions/:competition_id/report/:id(.:format)                          delegate_reports#update
                                                  PUT    /competitions/:competition_id/report/:id(.:format)                          delegate_reports#update

```

desired:

```
                             delegate_report_edit GET    /competitions/:competition_id/report/edit(.:format)                         delegate_reports#edit
                                  delegate_report GET    /competitions/:competition_id/report(.:format)                              delegate_reports#show
                                                  PATCH  /competitions/:competition_id/report(.:format)                              delegate_reports#update

```